packages/core/src/components/Shadows/ShadowColorPicker.tsx

```diff
@@ -0,0 +1,158 @@
+import {
+  Box,
+  Flex,
+  Slider,
+  SliderFilledTrack,
+  SliderThumb,
+  SliderTrack,
+  Text,
+} from '@chakra-ui/react'
+import { TShadowData } from '@core/types'
+import { RgbColor, SketchPicker } from '@hello-pangea/color-picker'
+import React, { useEffect, useState } from 'react'
+
+export function ShadowColorPicker({
+  variant,
+  setVariant,
+  presetColor,
+  initialValues,
+}: {
+  setVariant: React.Dispatch<React.SetStateAction<TShadowData>>
+  variant: {
+    name: string
+    value: string
+  }
+  presetColor: string
+  initialValues:
+    | {
+        hOffset: number
+        vOffset: number
+        blur: number
+        spread: number
+      }
+    | undefined
+}) {
+  const [color, setColor] = useState(
+    presetColor ? presetColor : 'rgba(1, 1, 1, 0.4)'
+  )
+
+  initialValues = initialValues
```

Comment:
nit: While this works, probably shouldn't modify props. Can probably update the function signature instead to define a default value, e.g.

```
export function ShadowColorPicker({
   initialValues = { hOffset: 0, vOffset: 0, blur: 0, spread: 0 },
} : { 
  initialValues?: { hOffset: number, vOffset: number, blur: 0, spread: 0 }
}
```

Reply:
Good callout. Also, initial values should be an array, right? because you can have multiple values. For example:
<img width="660" alt="Screen Shot 2023-03-28 at 7 21 48 PM" src="https://user-images.githubusercontent.com/29822597/228409951-3b134b22-d8c1-4aa0-b615-83109f8e6eb2.png">

Reply:
<img width="796" alt="Screen Shot 2023-03-28 at 7 22 32 PM" src="https://user-images.githubusercontent.com/29822597/228410064-c0409f69-1218-496c-b2fe-da60a3c370cf.png">
But appears to be a bug that only first value is captured